lib/xdp_loader.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __XDP_LOADER_H_
+#define __XDP_LOADER_H_
+
+using namespace std;
+
+class XdpLoader {
+    public:
+        XdpLoader(): handle(nullptr) { }
+        ~XdpLoader();
+        int LoadFrmFile(const string filepath, const int ifindex);
+        int LoadFrmBuffer(const string_view buffer, const char *name);
+    private:
+        void *handle;
```

Comment:
XdpLoader is no longer needed, however in the future you should follow the Google naming convention for member variables. Specifically handle should be "handle_" ending with an underscore.

Reply:
Done